src/api/spec/db/data/convert_notifications_event_payload_to_json_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+require Rails.root.join('db/data/20170831143534_convert_notifications_event_payload_to_json.rb')
+
+RSpec.describe ConvertNotificationsEventPayloadToJson do
```

Comment:
Nice! That's our first migration spec, right?

Reply:
I think so!